contracts/ronin/validator/CoinbaseExecution.sol

```diff
@@ -131,19 +144,26 @@ abstract contract CoinbaseExecution is
    *
    */
   function _syncBridgeOperatingReward(uint256 _lastPeriod, address[] memory _currentValidators) internal {
-    uint256 _totalBridgeBallots = _bridgeTrackingContract.totalBallots(_lastPeriod);
-    uint256 _totalBridgeVotes = _bridgeTrackingContract.totalVotes(_lastPeriod);
-    uint256[] memory _bridgeBallots = _bridgeTrackingContract.getManyTotalBallots(
+    IBridgeTracking bridgeTracking = _bridgeTrackingContract;
+
+    uint256 _totalBridgeBallots = bridgeTracking.totalBallots(_lastPeriod);
+    uint256 _totalBridgeVotes = bridgeTracking.totalVotes(_lastPeriod);
+    uint256[] memory _bridgeBallots = bridgeTracking.getManyTotalBallots(
       _lastPeriod,
       getBridgeOperatorsOf(_currentValidators)
     );
 
     if (
       !_validateBridgeTrackingResponse(_totalBridgeBallots, _totalBridgeVotes, _bridgeBallots) || _totalBridgeVotes == 0
     ) {
+      uint256 totalBridgeReward = _totalBridgeReward;
       // Shares equally in case the bridge has nothing to vote or bridge tracking response is incorrect
-      for (uint256 _i; _i < _currentValidators.length; _i++) {
-        _bridgeOperatingReward[_currentValidators[_i]] = _totalBridgeReward / _currentValidators.length;
+      for (uint256 _i; _i < _currentValidators.length; ) {
+        _bridgeOperatingReward[_currentValidators[_i]] = totalBridgeReward / _currentValidators.length;
+
+        unchecked {
+          ++_i;
+        }
```

Comment:
I think the new code will save tx gas, but increase the code size. Please have a small lab to test if using temp var will save both tx gas and code size. If it actually saves both, esc the code size, just keeping this pattern in the future impl.

Reply:
reverting to old code since this modification in fact increases code size.